ops-bedrock/docker-compose.yml

```diff
@@ -48,29 +49,7 @@ services:
       context: ../
       dockerfile: ./op-node/Dockerfile
     command: >
```

Comment:
I wonder why this was changed.

Reply:
not sure, I think my auto-formatter has some weird configuration, I'll revert because this is a lot less clean